ui/keyboardMove/src/main.ts

```diff
@@ -122,9 +128,19 @@ export function ctrl(root: KeyboardMoveRootCtrl): KeyboardMove {
     hasSelected: () => cg.state.selected,
     confirmMove: () => (root.submitMove ? root.submitMove(true) : null),
     usedSan,
-    jump(plyDelta: number) {
-      root.userJumpPlyDelta && root.userJumpPlyDelta(plyDelta);
-      root.redraw();
+    arrowNavigate(arrowKey: ArrowKey) {
+      if (root.handleArrowKey) {
+        root.handleArrowKey?.(arrowKey);
+        return;
+      }
+
+      const arrowKeyToPlyDelta = {
+        ArrowUp: -999,
+        ArrowDown: 999,
+        ArrowLeft: -1,
+        ArrowRight: 1,
+      };
+      root.userJumpPlyDelta?.(arrowKeyToPlyDelta[arrowKey]);
```

Comment:
There's an opportunity here to just remove `root.userJumpPlyDelta` in favor of `root.handleArrowKey`. I was debating whether it would be cleaner or not 🤔